releasenotes/notes/0.18/latex_style_user_math_text-266f9b411317ae6b.yaml

```
@@ -0,0 +1,31 @@
+---
+features:
+  - |
+    When using the ``latex`` or ``latex_source`` output mode of
+    :meth:`~qiskit.visualization.circuit_drawer` or the
+    :meth:`~qiskit.circuit.QuantumCircuit.draw` of
+    :class:`~qiskit.circuit.QuantumCircuit` the ``style`` kwarg
+    can now be used just as with the ``mpl`` output formatting.
+    However, unlike the ``mpl`` output mode only the ``displaytext``
+    field will be used when using the ``latex`` or ``latex_source`` output
+    modes (because neither supports color).
+  - |
+    When using the ``mpl`` or ``latex`` output methods for the
+    :meth:`~qiskit.visualization.circuit_drawer` function or the
+    :meth:`~qiskit.circuit.QuantumCircuit.draw` of
+    :class:`~qiskit.circuit.QuantumCircuit` you can now use math mode
+    formatting for text and additionally for color formatting
+    with the ``mpl`` drawer by using setting the ``style`` kwarg as a dict
+    with a user-generated name or label. For example, to add subscripts and to
+    change a gate color,
```

```suggestion
    change a gate color::
```

---

This actually should be

```suggestion
    change a gate color:
```

two `:` will be interpreted as a code block which will undo the jupyter execute directive